### ISSUE TITLE
Adjusting the key does not prevent CMD+L from locking the screen.

### ISSUE THREAD

**Issue by WardsParadox** [ftiff/MenuLock#2]
I used the menu to change the key to F19. Pressing CMD+L (to bring up munki's log) results in the screen still locking. Love the little menu app to add a functionality I personally needed 👍

**Comment by ftiff** [comment]
Thanks for your bug report!

Oh… yes I don't unregister the global shortcut. I'll correct this in the next release.
Meanwhile you can quit and restart the application!

Sorry and thanks again!

**Comment by WardsParadox** [comment]
Yup just figured that out! Thanks for confirming it though. Just grateful you created this app in the first place!
